recipe/patches/0035-do-not-override-win-sdk.patch

```diff
@@ -0,0 +1,28 @@
+From 710f54918fd44eca1522bb8e2234aa3e8493ec39 Mon Sep 17 00:00:00 2001
+From: Charles Bousseau <cbousseau@anaconda.com>
+Date: Tue, 1 Oct 2024 15:56:25 +0200
+Subject: [PATCH] do not override win sdk
+
+The python build looks at the registry to find the latest win sdk installed.
+If that SDK is too recent, version 10.0.19041.0 is forced. 
```

Comment:
The string `10.0.19041.0` does not appear elsewhere in the patch.  Have I missed something?

Reply:
It's here https://github.com/AnacondaRecipes/python-feedstock/pull/164/files#diff-cb1d9598c966571d80c42c2a1467b9ea50dd01e7dd25447cb9d6051476191346R21-R22

Reply:
Huh, why didn't I see that?  _cleans glasses_